docs/blog/2018-08-30-gatsby-values/index.md

```diff
@@ -0,0 +1,167 @@
+---
+title: What kind of company will we be? A look at Gatsby’s mission and values.
+author: Jason Lengstorf
+date: 2018-08-30
+image: images/values-whiteboard.jpg
+showImageInArticle: false
+tags: ["company"]
+---
+
+Most of the Gatsby [Inkteam](https://www.gatsbyjs.com/about/) was together in Portland last week for our latest team onsite, which we lovingly refer to as “Gatsby Gatherings”. The team assembles every few months to discuss pressing needs, our company’s direction, and our roadmap.
+
+This was our third Gathering, and it marked the first time when we weren’t in the midst of a major push (our first Gathering was centered around understanding how the Inkteam would actually function; the second around [officially launching the company](/blog/2018-05-24-launching-new-gatsby-company/)).
+
+But this time, with [the v2 release candidate launching last Tuesday](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/CHANGELOG.md#200-rc0-2018-08-21) and [Gatsby Preview in alpha](/blog/2018-07-17-announcing-gatsby-preview/), we had a chance to step back from immediate needs and start looking at the bigger picture: **what does Gatsby Inc. stand for? What are our goals and values?**
+
+This has been a _huge_ `TODO` on our list since the very beginning, so we spent a full day thinking through this, writing down our ideas, and discussing what kind of company we want to be.
+
+So last week, given the opportunity to spend in-person time together and talk about it, we tackled the existential questions behind Gatsby Inc.
+
+In this post, we’ll break down the process we followed, what we came up with, and how these decisions will impact Gatsby moving forward.
+
+## The process of defining company values
+
+Gatsby is a distributed team, which means that on most days we only interact with contributors and team members through Slack and GitHub messages. And with team members everywhere from San Francisco to Chiang Mai, getting the whole team on extended video calls to discuss philosophical questions is extremely difficult.
+
+<figure>
+  <img alt="The Gatsby team gathered around a whiteboard." src="./images/team-discussion.jpg" />
+  <figcaption>
+    The Inkteam in Portland, Oregon for a Gatsby Gathering. Photo:
+    <a href="https://www.houseofschaab.photography/">
+      Danielle Schaab
+    </a>
+  </figcaption>
+</figure>
+
+We took full advantage of being in the same room with each other to dig into these more nuanced discussions, which started a couple months ago when [Shannon](https://twitter.com/shannonb_ux) asked, “Do we have any company values?”
```

Comment:
Haha I do remember being pretty excited in my mind about values several times, and had forgotten that I brought it up! Glad it made a difference!